### ISSUE TITLE
mrna comparison gives "No data result" when any group misses data

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#7508]
Try: https://www.cbioportal.org/comparison/mrna?sessionId=5d1041b5e4b0ab41378771ff

![Screen Shot 2020-05-15 at 11 26 25 AM](https://user-images.githubusercontent.com/1334004/82067692-0ff67d00-969f-11ea-87c4-1f53379c32b0.png)

Works fine after unselecting NA group:

![Screen Shot 2020-05-15 at 11 28 07 AM](https://user-images.githubusercontent.com/1334004/82067798-30263c00-969f-11ea-82da-5eb1d78763ce.png)

Maybe we can filter all groups with only NA values by default and give some warning that certain groups had NA values?

**Comment by jjgao** [comment]
Thanks, @inodb!

@kalletlak have you fixed something similar to this, when there is no data at all in one group?

**Comment by kalletlak** [comment]
@jjgao yes, it was for alteration enrichments https://github.com/cBioPortal/cbioportal/issues/7010

**Comment by kalletlak** [comment]
@jjgao should I ignore just the groups which doesn't have alterations?. Here is the check https://github.com/cBioPortal/cbioportal/blob/3744576fd8179368f7896703729c3b408e0b0cda/service/src/main/java/org/cbioportal/service/impl/ExpressionEnrichmentServiceImpl.java#L116
